frontend/src/metabase/components/AdminPaneLayout.jsx

```diff
@@ -12,28 +14,33 @@ const AdminPaneTitle = ({
   buttonDisabled,
   buttonLink,
   headingContent,
-}) => (
-  <section className="clearfix px2">
-    {buttonText && buttonLink && (
-      <Link to={buttonLink} className="inline-block float-right">
-        <Button primary>{buttonText}</Button>
-      </Link>
-    )}
-    {buttonText && buttonAction && (
-      <Button
-        className="float-right"
-        primary={!buttonDisabled}
-        disabled={buttonDisabled}
-        onClick={buttonAction}
-      >
-        {buttonText}
-      </Button>
-    )}
-    {headingContent && <React.Fragment>{headingContent}</React.Fragment>}
-    {title && <h2 className="PageTitle">{title}</h2>}
-    {description && <p className="text-measure">{description}</p>}
-  </section>
-);
+}) => {
+  const buttonClassName = "ml-auto flex-no-shrink";
```

Comment:
- `ml-auto` (margin-left: auto) for pushing the button to the right of the screen
- `flex-no-shrink` (flex-shrink: 0) to not allow the button text to wrap